### DISCUSSION TITLE
No files getting selected when I perform migration

### DISCUSSION THREAD

**Discussion by Sakthivel185** [webdriverio/codemod#112] [Migration Issue]
Hi All,

I am trying to migrate my framework from Protractor to WebDriverIO 

Based on the migration guide from https://webdriver.io/docs/protractor-migration/#install-webdriverio-dependencies

I have install all the dependencies and try to migrate the config file using the below command

npx jscodeshift -t ./node_modules/@wdio/codemod/protractor --parser=tsx ./conf.ts

But no files are selected 
I have tried with absolute path also but still same

npx jscodeshift -t ./node_modules/@wdio/codemod/protractor --parser=tsx ./ 
if i give above command by giving only the root directory the command i working fine
But for individual file it is not working

I have attached the project structure for reference, can someone help me to resolve this issue

<img width="235" alt="image" src="https://github.com/webdriverio/codemod/assets/60173103/afece2e4-1cb2-456b-8308-bbf742b91669">
<img width="621" alt="image" src="https://github.com/webdriverio/codemod/assets/60173103/bab2df6e-31f8-4b3f-ab06-53a2bccbe9bb">

**Comment by christian-bromann** [comment]
This might be a limitation to `jscodeshift`. This needs further investigation though. I think in long term we might want to build our own CLI that takes over the file search.
